core/src/lib.rs

```diff
@@ -99,12 +103,22 @@ pub type DefaultFileService = FileServiceImpl<
 
 #[derive(Debug, Serialize)]
 pub enum InitLoggerError {
+    OnTouch(String),
     Unexpected(String),
 }
 
 pub fn init_logger(log_path: &Path) -> Result<(), InitLoggerError> {
-    loggers::init(log_path, env::var("LOCKBOOK_DEBUG").is_ok())
-        .map_err(|err| InitLoggerError::Unexpected(format!("{:#?}", err)))
+    let print_debug = env::var("LOCKBOOK_DEBUG").is_ok();
+    let print_colors = env::var("LOCKBOOK_NO_COLOR").is_err();
```

Comment:
Do you personally want this? I can't think of a use case where this is useful, especially because we have the file logging going. I'd rather just not have unless you can justify it's existence.

Reply:
Xcode logs don't support color but they do stdout

Reply:
That does now make me thing I can tail the file logs for color, but still I'm not going to completely debilitate my ability to use the in editor logs